pykrige/ok.py

```diff
@@ -404,6 +404,34 @@ def display_variogram_model(self):
                                         self.lags), 'k-')
         plt.show()
 
+    def get_variogram_points(self):
+        """Obtain the data pairs for the variogram model.
+           returns a numpy.array with the form [lag, variogram_value_at_lag]
+        """
+        variogram_points = self.variogram_function(self.variogram_model_parameters, self.lags)
+
+        return np.array([self.lags, variogram_points])
+
+    def save_variogram_to_file(self, output_file, separator=" "):

```

Comment:
I'm not so keen on this addition.

Assuming the above changes are made, this could be done fairly easily with,
```py
import pandas as pd

df = pd.DataFrame({'lag': model.lag,
                   'variogram_points': model.get_variogram_points()})
df.to_csv('<some-file>)
```
there is not need to include and maintain a custom CSV like format in pykrige for this..

Reply:
You are right, i'll take it out